plugins/auto_merge/auto_merge.go

```diff
@@ -1,26 +1,34 @@
 package auto_merge
 
 import (
+	"encoding/json"
 	"fmt"
 	"time"
 
+	"github.com/GEPROG/lassie-bot-dog/config"
+	autoMergeConfig "github.com/GEPROG/lassie-bot-dog/plugins/auto_merge/config"
```

Comment:
Why do you need to import something from the same package?

Reply:
It is not the same package. The package of this file is `auto_merge` the imported one is `config` which is a sub-package of `auto_merge`